MAUI/Expander/getting-started.md

```diff
@@ -11,6 +11,9 @@ documentation: ug
 
 This section provides a quick overview of how to get started with the SfExpander for .NET MAUI. Walk-through the entire process of creating the real world of this control.
 
+To get start quickly with .NET MAUI Expander, you can check on this video:
+<style>#MAUIExpanderVideoTutorial{width : 90% !important; height: 400px !important }</style> <iframe id='MAUIExpanderVideoTutorial' src='https://www.youtube.com/watch?v=zTVbps0m8i0'></iframe>
```

Comment:
do not use "watch keyword" link, 
use "embed" link. Refer ListView GettingStarted page

Reply:
updated